### ISSUE TITLE
Add data view for test

### ISSUE THREAD

**Issue by jugaldb** [vinitshahdeo/COVID19#13]
Columns for tests in both total data, statewise and the analytical charts would be a great addition

**Comment by Jayant0jha** [comment]
@jugaldb can you pls explain the requirements in detail here. I am intereseted to work on this.

**Comment by jugaldb** [comment]
Yes sure @Jayant0jha , what I want as an addition is to show the amount of tests done in various states and statistics according it which would display how the pandemic has grown along with the tests
